serverscommands/flavorcommands/get.go

```diff
@@ -40,5 +41,12 @@ func commandGet(c *cli.Context) {
 
 func tableGet(c *cli.Context, i interface{}) {
 	keys := []string{"ID", "Name", "Disk", "RAM", "RxTxFactor", "Swap", "VCPUs"}
-	output.MetaDataTable(c, i, keys)
+	f := func() map[string]interface{} {
+		return singleFlavor(i)
+	}
```

Comment:
True, that'd work, too. I guess my thinking was that since servers and potentially other resources needs it's own function to extract data, that each resource having a function to do that would be more consistent. I'll change it to see how it looks.